tests/system_tests_http2.py

```diff
@@ -548,7 +548,7 @@ def setUpClass(cls):
         cls.router_qdra.wait_router_connected('QDR.B')
         cls.router_qdrb.wait_router_connected('QDR.A')
 
-        sleep(2)
+        sleep(1)
```

Comment:
I recall some tests were sending a probe message through the entire router network to check it is established... or they were checking qdstat. Afaik the preferred way was the probe message.

Reply:
This sleep is there because there is really no way to find out if the connector made a connection to the http2 server. But now that I think about it, the http2 server connection is the first connection that the router attempts and then it moves to the establish the inter-router connector. I am taking out these sleeps.